notebooks/classification/mcmc_bnn_makemoons.ipynb

```diff
@@ -0,0 +1,462 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    from flax.core import unfreeze
</pre><p>There seem to be two unfreeze imports.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nipunbatra/bnn-zoo/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>